.travis.yml

```diff
@@ -31,6 +31,14 @@ jobs:
       - TESTING_PROFILE=unit-tests
       script: mvn --batch-mode clean install -P$TESTING_PROFILE
 
+    - stage: automated-review
+      jdk: openjdk10
```

Comment:
Should we be running locally with Open JDK 10? I'm still using Java 8 in my local dev environment.

Reply:
I've been slipping in openjdk 10 stuff here and there ahead of a formal switch. 
I do want to switch long term from oracle 8 to openjdk 10/11